studio/settings.py

```diff
@@ -387,3 +393,72 @@
 # Specifically, apps.tests.test_user_has_no_access was failing.
 # Also anonymous access to pages was not working.
 ANONYMOUS_USER_NAME = None
+
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "json_formatter": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.processors.JSONRenderer(),
+        },
+        "plain_console": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.dev.ConsoleRenderer(),
+        },
```

Comment:
So I think we should choose between json renderer and console renderer based on DEBUG value.
Or just go for json. But these messages are nicer when one is developing locally 
<img width="710" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/953c69b9-7fb4-4405-b58c-b9129d0b40d9">